vng_api_common/management/commands/patch_error_contenttypes.py

```diff
@@ -21,11 +21,9 @@ def add_arguments(self, parser):
 
     def patch_response(self, response):
         content = {}
-
```

Comment:
Deze wijzigingen in dit bestand zou ik achterwegen laten, ze voegen niet veel toe